### ISSUE TITLE
Stopped working at EXPO SDK 43

### ISSUE THREAD

**Issue by nikopolidi** [kimxogus/react-native-version-check#149]
Expo has deprecated unimodules and moved to 'expo' package;
https://blog.expo.dev/expo-sdk-43-aa9b3c7d5541

`unable to resolve
> 1 | import { NativeModulesProxy } from '@unimodules/core';
`

Migrating: https://github.com/expo/fyi/blob/master/expo-modules-migration.md

**Comment by liorokach** [comment]
Yes, with the migration from @unimodules now there is a problem..
Should be changed to 
`import { NativeModulesProxy } from 'expo-modules-core'`

**Comment by victorglezcardoso** [comment]
> Yes, with the migration from @unimodules now there is a problem.. Should be changed to `import { NativeModulesProxy } from 'expo-modules-core'`

Is there a way of resolve this locally?

**Comment by cristianFryla** [comment]
@victorglezcardoso did you find a way to solved it locally as you asked? Tks!

**Comment by victorglezcardoso** [comment]
[cristianFryla](https://github.com/cristianFryla) No, unfortunately not. I make a walkaround controlling manually in the app the version of the store via a rest request to my server. When I update the app, I change in my server the version of the app and if it is necessary I force the app to update... Cheers.

**Comment by cristianFryla** [comment]
> [cristianFryla](https://github.com/cristianFryla) No, unfortunately not. I make a walkaround controlling manually in the app the version of the store via a rest request to my server. When I update the app, I change in my server the version of the app and if it is necessary I force the app to update... Cheers.

Ok, tks, I did the same unfortunaltely!!

**Comment by rf1804** [comment]
I personally don't use expo, happy to have a pr if someone can

**Comment by FatmaMahmoud698** [comment]
@rf1804 can you just upgrade the package used on folder related to the package react-native-version-check-expo

**Comment by rf1804** [comment]
> @rf1804 can you just upgrade the package used on folder related to the package react-native-version-check-expo

@FatmaMahmoud698 It was already done v3.4.3 was released 12 hours ago

https://www.npmjs.com/package/react-native-version-check-expo

**Comment by FatmaMahmoud698** [comment]
@rf1804 on this path [react-native-version-check](https://github.com/kimxogus/react-native-version-check)/[packages](https://github.com/kimxogus/react-native-version-check/tree/master/packages)/[react-native-version-check-expo](https://github.com/kimxogus/react-native-version-check/tree/master/packages/react-native-version-check-expo)/package.json
there are still old versions

**Comment by stri8ed** [comment]
@FatmaMahmoud698 

I created a fork, which does not depend on expo-localization. 

https://www.npmjs.com/package/expo-react-native-version-checker
